packages/node-cli/README.md

```diff
@@ -12,7 +12,7 @@ CLI for Node.js is an interactive tool that guides you through all the steps of
 
 ## Prerequisites
 The following software is required to work with SuiteCloud CLI for Node.js:
-- Node.js version 14 LTS or greater
+- Node.js version 14 LTS
```

Comment:
@x8a Can you change the version to 14.16.0 LTS

Reply:
Done!